api/job/janitor_test.go

```diff
@@ -3,17 +3,48 @@ package job
 import (
 	"testing"
 	"time"
+
+	"github.com/quintilesims/layer0/common/models"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestJanitor(t *testing.T) {
 	store := NewMemoryStore()
 	janitor := NewJanitor(store, time.Hour*24)
 
-	// todo: put some jobs in the store
+	expiry := -24 * time.Hour
```

Comment:
This doesn't make sense because we wouldn't have a negative expiry. It also makes the logic below, `now.Add(expiry * -2)`, really confusing. IMO, 'keep' should just `expiry - 1` and 'delete' should be `expiry + 1`; that make it very clear we keep things that are just before expiring, and delete things just after expiring.

Reply:
I agree the logic doesn’t make sense. I tried to debug the issue, because when I do addition it doesn’t change the value. I tried wrapping it in ‘( )’ but it didn’t change anything. Have you seen this?